src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public enum Colors {
+        white,
```

Comment:
read about naming convention for enum 
```suggestion
        WHITE,
```